### DISCUSSION TITLE
Stream the birdseye view?

### DISCUSSION THREAD

**Discussion by simeon-walker** [blakeblackshear/frigate#2263] [Ask A Question]
Hi,
Is there a way to stream the birdseye view? I'd like to be able to access it from another machine, e.g. a Pi running VLC.
Thanks.

**Comment by DrSpaldo** [comment]
I have been looking at the same thing. Is there a RTMP link or similar to see the Birdseye?

**Reply by NickM-27** [reply]
If viewing the jsmpeg stream in something like the frigate HASS card, using birdseye as the camera name is supported.

**Reply by Davy-H** [reply]
Hi @NickM-27 Do you have to do something special to get it? I don't have it have it as a option in home assistant.

**Reply by NickM-27** [reply]
You gotta use the hass frigate card and insert a camera with birdseye as the name.

**Reply by Davy-H** [reply]
I don't have a camera called birdseye getting reported. All my real cameras are there just not birds eye. Does it depend on the version of frigate (I'm on 10rc1) or home assistant frigate integration (I'm on Mar 7 git commit)?

**Reply by NickM-27** [reply]
@Davy-H You have to manually type it in as the name inside the frigate card, it won't be a selectable camera entity.

**Reply by Davy-H** [reply]
Thanks. That worked :)
